config.go

```diff
@@ -121,6 +130,41 @@ type CORSConfig struct {
 	//
 	// Added in v4.2.0.
 	AllowAllOrigins bool
+
+	// AllowOrigins enables CORS and allows the list of origins in the
+	// HTTP request origins when set. Practically speaking, this
+	// results in the HTTP header "AllowOrigins".
+	//
+	// Added in v5.0.0.
+	AllowOrigins string
+}
+
+// OIDCConfig holds settings for the HTTP server's OIDC access token validation settings.
+type OIDCConfig struct {
+
+	// Enable this functions as a switch to enable or disable the validation of OIDC
+	// access bearer tokens.
+	//
+	// Added in v5.0.0.
+	Enable bool
```

Comment:
The first word should be name of the field, which it is here.

But it's still a part of the sentence.

```suggestion
	// Enable functions as a switch to enable or disable the validation of OIDC
	// access bearer tokens.
	//
	// Added in v5.0.0.
	Enable bool
```

Similar thing with the other godocs, such as `IssuerURL an integral part...` &rarr; `IssuerURL is an integral part...`